packages/select/src/components/query-list/queryList.tsx

```diff
@@ -262,6 +262,9 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
         const maybeNoResults = this.isCreateItemRendered() ? null : noResults;
         const menuContent = renderFilteredItems(listProps, maybeNoResults, initialContent);
         const createItemView = this.isCreateItemRendered() ? this.renderCreateItemMenuItem(this.state.query) : null;
+        if (menuContent === null && createItemView === null) {
```

Comment:
I think you should check `== null` in both cases. we don't have strict null checks enabled in this project. `maybeNoResults` could be `undefined` at this point.

Reply:
yes `==` is essential here. looks good otherwise.